# DP: [AArch64] Fix output template for Scalar Neon->Neon register move

https://gcc.gnu.org/ml/gcc-patches/2013-10/msg01197.html

--- a/src/gcc/config/aarch64/aarch64.md
+++ b/src/gcc/config/aarch64/aarch64.md
@@ -608,7 +608,7 @@
      case 8:
        return "dup\t%0.<Vallxd>, %w1";
      case 9:
-       return "dup\t%0, %1.<v>[0]";
+       return "dup\t%<Vetype>0, %1.<v>[0]";
      default:
        gcc_unreachable ();
      }
